x/mock/simulation/rand_util.go

```diff
@@ -38,37 +37,33 @@ func RandStringOfLength(r *rand.Rand, n int) string {
 }
 
 // Generate a random amount
-// Note: The range of RandomAmount includes max, and is, in fact, biased to return max.
+// Note: The range of RandomAmount includes max, and is, in fact, biased to return max as well as 0.
 func RandomAmount(r *rand.Rand, max sdk.Int) sdk.Int {
```

Comment:
```suggestion
func BiasedRandomAmount(r *rand.Rand, max sdk.Int) sdk.Int {
```

Reply:
I don't think this is necessary to differentiate... RandomAmount might as well be biased all the time intelligently...

Reply:
IMHO it's not intuitive. Can the same be said for the native go library's rand functions?